subsys/bluetooth/audio/bap_scan_delegator.c

```diff
@@ -87,7 +87,16 @@ struct bt_bap_scan_delegator_inst {
 		[CONFIG_BT_BAP_SCAN_DELEGATOR_RECV_STATE_COUNT];
 };
 
-static bool conn_cb_registered;
+enum scan_delegator_registered_flags {
+	REGISTERED_CONN_CB,
+	REGISTERED_SCAN_DELIGATOR,
+	REGISTERED_PA_SYNC_CB,
+
+	REGISTERED_FLAGS_NUM,
```

Comment:
I strongly prefer that the enum name is prefix in the values:
```suggestion
enum scan_delegator_registered_flags {
	SCAN_DELEGATOR_REGISTERED_CONN_CB,
	SCAN_DELEGATOR_REGISTERED_SCAN_DELIGATOR,
	SCAN_DELEGATOR_REGISTERED_PA_SYNC_CB,

	SCAN_DELEGATOR_REGISTERED_FLAGS_NUM,
```

Consider also just renaming this to `scan_delegator_flag` so that it can be used for non-register things in the future

Reply:
Ok.